src/local.rs

```diff
@@ -316,3 +390,21 @@ pub fn confirm_continue() -> bool {
     std::io::stdin().read_exact(&mut buffer).unwrap();
     matches!(buffer[0], b'y' | b'Y')
 }
+
+pub async fn delete(app_instance: &str, docker: bool, dry_run: &Option<DryRun>) -> Result<()> {
+    match dry_run {
+        Some(DryRun::Render | DryRun::Diff) => {
+            Err(Error::UnsupportedDryRunOption(dry_run.clone().unwrap()).into())
+        }
```

Comment:
Perhaps there's a more idiomatic Rust way to do this, but the `--dry-run diff|render` aren't implemented on this `delete` command yet, so instead of it dropping to some form of undefined behaviour, I think it's better to come back early and say its unsupported at the moment.

For `--dry-run render`, I don't think it makes sense anyway because we have no `kubecfg show` to see. A `render` would really just be either nothing (because everything is getting deleted) or maybe the generated `ConfigMap` (in which case its blank so probably not necessary).

I think the `diff` falls into the same category as well, perhaps it's more useful though for "show me what will be deleted in a nice view", but I don't want to sprawl a tonne of things into this PR as well :smile:

Reply:
Yes I'm fine with not implementing diff  for delete for now